packages/esm-patient-vitals-app/src/vitals/vitals-header/vital-header-state.component.tsx

```diff
@@ -44,14 +45,14 @@ const VitalHeader: React.FC<VitalHeaderProps> = ({ patientUuid, showRecordVitals
   ] = conceptsUnits;
 
   useEffect(() => {
-    if (patientUuid) {
-      const subscription = performPatientsVitalsSearch(config.concepts, patientUuid, 10).subscribe((vitals) => {
+    if (patientUuid && concepts) {
+      const subscription = performPatientsVitalsSearch(concepts, patientUuid, 10).subscribe((vitals) => {
         setVital(vitals[0]);
         setIsLoading(false);
       }, createErrorHandler);
       return () => subscription.unsubscribe();
     }
-  }, [patientUuid, config.concepts]);
+  }, [patientUuid]);
```

Comment:
Still, if we changed concepts we should re-fetch.

My question is; why did concepts change in the first place?

Reply:
Concepts changed because the `useConfig` updates the concepts. I will close this PR since the best place to make the changes is on the `useConfig` hook to return values that don't change over re-renders